x/metadata/client/cli/cli_test.go

```diff
@@ -213,13 +213,13 @@ func (s *IntegrationCLITestSuite) SetupSuite() {
 	s.contractSpecID = metadatatypes.ContractSpecMetadataAddress(s.contractSpecUUID)
 	s.recordSpecID = metadatatypes.RecordSpecMetadataAddress(s.contractSpecUUID, s.recordName)
 
-	s.scope = *metadatatypes.NewScope(
-		s.scopeID,
-		s.scopeSpecID,
-		ownerPartyList(s.user1AddrStr),
-		[]string{s.user1AddrStr},
-		s.user2AddrStr,
-	)
+	s.scope = metadatatypes.Scope{
```

Comment:
I'm really not a fan of the `NewScope` function.

1. It doesn't do anything that direct struct instantiation doesn't do.
2. It's less readable since you can't use named parameters when calling functions.
3. If this had already used struct initialization (instead of `NewScope`), I wouldn't have had to update these lines at all.

In all these changes, specially in the unit tests, I favored switching to struct initialization, but I left uses of `NewScope` where switching would have added a bunch of lines.  E.g. In this case, no new lines needed to be added, and switching to struct initialization actually saved an extra line because I didn't need to provide the new `false`. But in some of the other unit tests where the `NewScope` and args are all on one line, I just added the `, false`.

Ideally, it saves us time the next time an argument needs to be added to a scope; we don't have to update tests that continue to pass using the zero-value of the new field.